Example/Example for Delirium/Modules/Pin/CustomPinViewController.swift

```diff
@@ -9,33 +9,31 @@
 import UIKit
 import Delirium
 
+@available(iOS 9, *)
 class CustomPinViewController: UIViewController {
+
+    // MARK: - Outlets
+
+
+    @IBOutlet weak var pinView: PinView!
     
     // MARK: - View flow
     
     override func viewDidLoad() {
         super.viewDidLoad()
         
         title = "Pin Code"
-    }
-    
-    // MARK: - Actions
-    
-    @IBAction func setupPinCode(_ sender: AnyObject) {
-        let configuration = PinConfiguration()
-        let controller = pinViewController(withConfiguration: configuration)
-        controller.delegate = self
-        navigationController?.pushViewController(controller, animated: true)
+
+        pinView.delegate = self
     }
     
 }
 
-extension UIViewController: PinViewControllerDelegate {
-    
-    public func pinViewController(_ controller: PinViewController, didEnterPin pin: String) {
-        let controller = UIAlertController(title: "Pin code", message: "The pincode you entered is \(pin).", preferredStyle: .alert)
-        controller.addAction(UIAlertAction(title: "Ok", style: .default, handler: nil))
-        present(controller, animated: true, completion: nil)
+@available(iOS 9, *)
+extension CustomPinViewController: PinViewDelegate {
+
+    func pinView(view: PinView, didEnterPin pin: String) {
+        print("Entered the correct pin", pin == "1234")
```

Comment:
Het ding is dat ik Delirium eigenlijk geen dependencies wil geven. Dat dat echt op zichzelf gebruikt kan worden.